e2e/test/provisioning/ReprovisioningE2ETests.cs

```diff
@@ -1,553 +0,0 @@
-﻿// Copyright (c) Microsoft. All rights reserved.
-// Licensed under the MIT license. See LICENSE file in the project root for full license information.
-
-using System;
-using System.Collections.Generic;
-using System.Diagnostics.Tracing;
-using System.Net;
-using System.Security.Cryptography.X509Certificates;
-using System.Text;
-using System.Threading;
-using System.Threading.Tasks;
-using Microsoft.Azure.Devices.Client;
-using Microsoft.Azure.Devices.Provisioning.Client;
-using Microsoft.Azure.Devices.Provisioning.Client.Transport;
-using Microsoft.Azure.Devices.Provisioning.Security.Samples;
-using Microsoft.Azure.Devices.Provisioning.Service;
-using Microsoft.Azure.Devices.Shared;
-using Microsoft.VisualStudio.TestTools.UnitTesting;
-using static Microsoft.Azure.Devices.E2ETests.Provisioning.ProvisioningE2ETests;
-using static Microsoft.Azure.Devices.E2ETests.Provisioning.ProvisioningServiceClientE2ETests;
-
-namespace Microsoft.Azure.Devices.E2ETests.Provisioning
-{
-    [TestClass]
-    [TestCategory("E2E")]
-    [TestCategory("DPS")]
-    [TestCategory("LongRunning")]
-    public class ReprovisioningE2ETests : E2EMsTestBase
```

Comment:
Why is this being deleted?

Reply:
It no longer exists in main. Correct me if I am wrong, I heard ReprovisioningE2ETests are testing the server side, not the SDK.